src/main/resources/add-ons-to-index.json

```diff
@@ -407,7 +422,7 @@
       "type": "OMOD",
       "name": "chirdlutil",
       "tags": [
-        "chica"
+        "chica", "chirdlutil"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.